### PR TITLE
Update pl_pl.json

### PR DESCRIPTION
**Pull request by kotelkonrad**

### CODE REVIEW

**Comment by Yamahari** [comment]
wood only variants are no longer called x_wooden_x_axe etc. instead only the wood type, e.g. oak_axe instead of oak_wooden_oak_axe as there is no need for that anymore

**Comment by kotelkonrad** [comment]
this don't have x_wooden_x

File: src/main/resources/assets/ilikewood/lang/pl_pl.json [hunk 1]
```diff
@@ -77,4 +77,154 @@
...
+  "item.ilikewood.golden_oak_axe": "Złota dębowa siekiera",
+  "item.ilikewood.oak_shovel": "Dębowa łopata",
+  "item.ilikewood.diamond_acacia_shovel": "Diamentowa akacjowa łopata",
+  "item.ilikewood.dark_oak_axe": "Ciemnodębowa siekiera",
+  "item.ilikewood.iron_oak_pickaxe": "Żelazny dębowy kilof",
+  "item.ilikewood.dark_oak_shovel": "Ciemnodębowa łopata",
+  "item.ilikewood.golden_dark_oak_hoe": "Złota ciemnodębowa motyka",
+  "item.ilikewood.golden_dark_oak_shovel": "Złota ciemnodębowa łopata",
+  "item.ilikewood.diamond_jungle_axe": "Diamentowa tropikalna siekiera",
+  "item.ilikewood.birch_wooden_birch_hoe": "Brzozowa motyka",
```
Comment (Yamahari) [inline src/main/resources/assets/ilikewood/lang/pl_pl.json]: Here for example, it should be item.ilikewood.birch_hoe

**Comment by kotelkonrad** [comment]
fixed

**Pull request merged**